lib/linguist/languages.yml

```diff
@@ -619,6 +619,14 @@ Brainfuck:
   codemirror_mode: brainfuck
   codemirror_mime_type: text/x-brainfuck
   language_id: 38
+BrighterScript:
+  type: programming
+  color: "#66AABB"
+  extensions:
+  - ".bs"
```

Comment:
🤔 this is going to result in a lot of files being incorrectly classified as there are a lot of `.bs` files which don't look anything like your samples. As this is the only instance of `.bs` known to Linguist, I think we're going to need to identify at least one of the other popular users of this extension and add support for it too.

Reply:
so another language that has .bs as file extension?

Reply:
Yup. It can be added as part of this PR.

Reply:
i didn't understand, the samples delivered are one from plain brightScript and then an example of what brighterScript bring to the table as a superset of brightScript, classes, ternary operators, etc.

I cannot provide real world usage of the language myself as our source code is private, but in the query search you have fine examples of brighterScript

Reply:
added Bikeshed language and heuristics.

please have a look when able. 
thank you

Reply:
@lildude can you have a look at this pr when you have some time please and let me know if anything missing? I believe with the adding of bikeshed and heuristics, it should be done right?